optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/list/externalized/TestdataListEntityExternalized.java

```diff
@@ -28,7 +28,7 @@
 public class TestdataListEntityExternalized extends TestdataObject {
 
     @PlanningListVariable(valueRangeProviderRefs = "valueRange")
-    private List<TestdataListValueExternalized> valueList;
+    public List<TestdataListValueExternalized> valueList;
```

Comment:
Per the comment: for the test to run on the GIZMO SolutionCloner, ALL fields must be public (this is because GIZMO is prohibited from using getters/setters since they may be complex/not simple.